arobit/values.yaml

```diff
@@ -0,0 +1,941 @@
+## Global Docker image parameters
+##
+# imagePullSecrets:
+#   - linuxgeneva-microsoft
+
+forwarder:
+  ## Enable forwarder daemonset
+  ##
+  enabled: true
+  dynamicConfig: false
+
+  ## Specifies whether the fluentbit storage metrics should be exported to prometheus format
+  ##
+  jsonExporter: false
+
+  ## Specifies whether mdsd should enabled
+  ##
+  mdsd: false
+
+  ## K8s Security Context for forwarder pods
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+  ##
+  securityContext:
+    enabled: true
+    runAsUser: 65532
+    runAsGroup: 65532
+
+  ## Set up update strategy.
+  ## ref: https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/#daemonset-update-strategy
+  ## Example:
+  # updateStrategy:
+  #  type: RollingUpdate
+  #  rollingUpdate:
+  #    maxSurge: 25%
+  #    maxUnavailable: 25%
+  updateStrategy:
+    type: RollingUpdate
+
+  ## Set Priority Class Name to allow priority control over other pods
+  ## ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+  ##
+  priorityClassName: ""
+
+  ## Node labels for pod assignment
+  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## Tolerations for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+
+  ## Affinity for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {}
+
+  ## Annotations for the forwarder
+  ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+  ##
+  annotations: {}
+
+  ## Pod annotations
+  ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+  ##
+  podAnnotations: {}
+
+  ## Extra labels to add to Pod
+  podLabels: {}
+
+  ## Pods Service Account
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+  serviceAccount:
+    ## Specifies whether a ServiceAccount should be created
+    ##
+    create: true
+    ## The name of the ServiceAccount to use.
+    ## If not set and create is true, a name is generated using the template
+    # name:
+    ## Annotations for the Service Account (evaluated as a template)
+    ##
+    annotations: {}
+
+  ## Role Based Access
+  ## ref: https://kubernetes.io/docs/admin/authorization/rbac/
+  ##
+  rbac:
+    create: true
+
+  ## Service parameters
+  ##
+  service:
+    ## Service ports
+    ##
+    ports:

```

Comment:
Add TCP input / output to forward auditlogs

https://github.com/microsoft/go-otel-audit/blob/24f0bd5eca07375fd52c22fa4ca004b49a587b41/audit/docs/highlevel.md#introduction